api/s2n.h

```diff
@@ -205,6 +205,8 @@ S2N_API
 extern int s2n_config_set_verification_ca_location(struct s2n_config *config, const char *ca_pem_filename, const char *ca_dir);
 S2N_API
 extern int s2n_config_add_pem_to_trust_store(struct s2n_config *config, const char *pem);
+S2N_API
+extern int s2n_config_wipe_trust_store(struct s2n_config *config);
```

Comment:
Can you add documentation to this function? Ideally describe what its usecase is (disabling the system trust store, etc)

You can see an example here: https://github.com/aws/s2n-tls/blob/aab3190ae86fdefe4b46c9689139bb7c57f94a38/api/s2n.h#L580-L599